tmt/utils/templates.py

```diff
@@ -503,17 +503,15 @@ def raise_error(message: str) -> None:
     try:
         return environment.from_string(template).render(**variables).strip()
 
-    except jinja2.exceptions.TemplateSyntaxError as exc:
+    except jinja2.exceptions.TemplateSyntaxError as error:
```

Comment:
Check the diff: at some places, `as exc` is still used while here I see a rename.